### PR TITLE
Add clearPolicyCache() 

### PR DESCRIPTION
**Pull request by giarc3**
Add clearPolicyCache() to IronOxide to clear the cached policies

### CODE REVIEW

**Comment by giarc3** [comment]
> It seems like a test that shows that clearPolicyCache is not always 0. We can clear after we evaluate a policy in another test.

I added a test that caches a policy and then clears it.

File: src/lib.rs.in [hunk 1]
```diff
@@ -852,6 +853,10 @@ foreigner_class!(
 class IronOxide {
     self_type IronOxide;
     private constructor = empty;
+    /// Clears all entries from the policy cache
+    /// 
+    /// @return number of entries cleared from the cache
+    method BlockingIronOxide::clear_policy_cache(&self) -> usize; alias clearPolicyCache;
```
Comment (clintfred) [inline src/lib.rs.in]: I think this should be 
``` method IronOxide::clear_policy_cache(&self) -> usize; alias clearPolicyCache;```
to remain consistent with the rename of `BlockingIronOxide` -> `IronOxide` that occurrs in `lib.rs`

Comment (giarc3) [reply src/lib.rs.in]: Changed

File: tests/src/test/scala/ironrust/FullIntegrationTest.scala [hunk 2]
```diff
@@ -609,7 +610,30 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
       result.getErrors.getGroups should have length 2
       result.getErrors.getGroups.head.getId.getId shouldBe "badgroupid_frompolicy"
       result.getErrors.getGroups.tail.head.getId.getId shouldBe s"data_recovery_${primaryTestUserId.getId}"
+    }
 
+    "clear an empty policy" in {
+      val data: Array[Byte] = List(1, 2, 3).map(_.toByte).toArray
+      val id = GroupId.validate(s"data_recovery_${primaryTestUserId.getId}")
```
Comment (coltfred) [inline tests/src/test/scala/ironrust/FullIntegrationTest.scala]: A comment as to why this id is important might be nice for the future.

Comment (giarc3) [reply tests/src/test/scala/ironrust/FullIntegrationTest.scala]: Added

**Pull request merged**